core/source/Http.mint

```diff
@@ -297,25 +315,72 @@ module Http {
         delete this._requests[#{uid}]
 
         resolve(#{Result::Err({
+          headers: `getResponseHeaders()`,
           type: Http.Error::Timeout,
           status: `xhr.status`,
           url: request.url
         })})
       })
 
-      xhr.addEventListener('load', (event) => {
+      xhr.addEventListener('load', async (event) => {
         delete this._requests[#{uid}]
 
+        let contentType = xhr.getResponseHeader("content-type");
```

Comment:
```suggestion
        let contentType = xhr.getResponseHeader("Content-Type");
```

Reply:
Nice catch :+1:

Reply:
It's purely stylistic, my ADHD-driven OCD couldn't not-notice it though :P

Reply:
Oh, I was wrong, it's case-insensitive :joy: Well if it makes you happy :grin: